zeth-contracts/contracts/Pairing.sol

```diff
@@ -70,20 +78,23 @@ library Pairing {
         input[2] = s;
         bool success;
         assembly {
-            // bn256ScalarMul precompiled: https://github.com/ethereum/go-ethereum/blob/master/core/vm/contracts.go#L58
-            // Gas cost: 40000 (see: https://github.com/ethereum/go-ethereum/blob/master/params/protocol_params.go#L85)
-            success := call(sub(gas, 2000), 7, 0, input, 0x80, r, 0x60)
+            // bn256ScalarMul precompiled:
+            //   https://github.com/ethereum/go-ethereum/blob/master/core/vm/contracts.go#L58
+            // Gas cost: 40000
```

Comment:
The lines have changed because of the instanbul update: 
https://github.com/ethereum/go-ethereum/blob/3bb6815fc1ade61e85b7039cf2e08a0871bb47e4/params/protocol_params.go#L126

we should also update to istanbul costs